source/guides/network/running-a-symbol-node-manually.rst

```diff
@@ -0,0 +1,194 @@
+
+Run the broker
+===================
+
+4. **Edit** ``resources/config-database.properties`` and point ``databaseUri`` to ``127.0.0.1:27017``:
+
+   .. code-block:: ini
+
+      [database]
+  
+      databaseUri = mongodb://127.0.0.1:27017
+      databaseName = catapult
+
+5. **Run the broker**:
+
+   Move into the ``bin`` directory and run:
+
+   .. code-block:: bash
+
+      ./catapult.broker ..
+
+   Keep the broker running for as long as your node runs.
+
+Build and run the REST gateway
+==============================
+
+6. **Clone** the ``catapult-rest`` repository and **install its dependencies**:
+
+   .. code-block:: bash
+
+      git clone https://github.com/nemtech/catapult-rest.git
+      cd catapult-rest
+      npm install -g yarn
+      ./yarn-setup.sh
+
+7. **Build** the REST gateway:
+
+   .. code-block:: bash
+
+      cd rest
+      yarn build
+
+8. **Configure** the REST gateway:
+
+   All configuration is stored in the ``resources/rest.json`` file.
+   
+   **Edit this file** to make sure that the following properties point to the right files:
+
+   .. csv-table::
+      :header: "Property", "File", "Default location"
+      :widths: 35,35,30
+      :delim: ;
+
+      ``tlsClientCertificatePath``; ``node.crt.pem``; ``_build/certificate/``
+      ``tlsClientKeyPath``; ``node.key.pem``; ``_build/certificate/``
+      ``tlsCaCertificatePath``; ``ca.cert.pem``; ``_build/certificate/``
+      ``networkPropertyFilePath``; ``config-network.properties``; ``_build/resources/``
+      ``nodePropertyFilePath``; ``config-node.properties``; ``_build/resources/``
+
+   For example:
+
+   .. code-block:: json
+
+      {
+         "host": "127.0.0.1",
+         "port": 7900,
+         "timeout": 1000,
+         "tlsClientCertificatePath": "~/catapult-server/_build/certificate/node.crt.pem",
+         "tlsClientKeyPath": "~/catapult-server/_build/certificate/node.key.pem",
+         "tlsCaCertificatePath": "~/catapult-server/_build/certificate/ca.cert.pem",
+         "networkPropertyFilePath": "~/catapult-server/_build/resources/config-network.properties",
+         "nodePropertyFilePath": "~/catapult-server/_build/resources/config-node.properties"
+      }
+
+9. **Run the REST gateway**:
+
+   .. code-block:: bash
+
+      node _build/index.js resources/rest.json
+
+   Keep the REST gateway running for as long as your node runs.
+
+Verification
+============
+
+If the REST gateway is working correctly, you can now make queries at port 3000. If the Catapult Server, Broker and MongoDB are working correctly, the values returned by REST will be correctly synchronized with the rest of the blockchain.
+
+Point a browser to the following URLs and check that the returned values are similar to the expected ones:
+
+* `http://localhost:3000/node/info <http://localhost:3000/node/info>`__
+
+  .. code-block:: json
+
+     {
+        "status":{
+           "apiNode":"up",
+           "db":"up"
+        }
+     }
+
```

Comment:
perhaps outdated?
I can't see any status info in the JSON I get from port 3000:

```
version | 655368
-- | --
publicKey | "EB6003935A7422EAEE04DDD4D7798E076DF672FDA6A3E37A234012C153969230"
networkGenerationHashSeed | "4DF76A205008026C776CB6AFD6D3393F04CD458E0DD2DDFDDDF31A402072B2D6"
roles | 65
port | 7900
networkIdentifier | 168
host | "192.168.0.38"
friendlyName | "nodish1"
nodePublicKey | "DB5FC52F0F0AF9D8FAF6BF37D4B9FF7CCCA389A59AA34609796576FBF09D255D"

```
?

Reply:
That does not look like JSON at all, did you really get it from http://localhost:3000/node/info ?